packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -447,15 +447,29 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         super(selectionService, handler);
     }
 
+    isVisible(): boolean {
+        return super.isVisible();
```

Comment:
I thought you want to some special checks here, if it just calls super then we call removed these methods all together